src/loader/endpoints/corrections.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd
+import Levenshtein as lev
+
+main_corrections = pd.DataFrame(
+    {
+        "city_name": ["Januário Cicco"],
+        "state_num_id": [24],
+        "correction_name": ["Boa Saúde"],
+    }
+)
+in_loco_corrections = pd.DataFrame(
+    {
+        "city_name": ["São Valério da Natividade", "Santa Teresinha", "Campo Grande"],
+        "state_name": ["Tocantins", "Bahia", "Rio Grande do Norte"],
+        "correction_name": ["São Valério", "Santa Terezinha", "Augusto Severo"],
+    }
+)
+
+# In: Name of the city, id of its state and the database over which to compare it
+# Out: the minimal Lev distance found and the closes word for it (within the same state always)
+def min_lev(city, state_id, target_df):
+    min_dis = float("inf")
+    word = None
+    for index, row in target_df.iterrows():
+        if state_id == row["state_num_id"]:  # if on the same state
+            dis = lev.distance(city, row["city_name"])
+            if dis < min_dis:
+                word = row["city_name"]
+                min_dis = dis
+    return (min_dis, word)
+
+
+# IN: cities dataframe from main
+# OUT cleaned version of it
+def correct_main_cities(cities_df):
+    cities_df_main = cities_df

```

Comment:
Why to assign a new variable?

Under the hood, what you are doing is creating a pointer from cities_df to cities_df_main.